utils/spacewalk-common-channels.ini

```diff
@@ -1332,6 +1332,24 @@ checksum = sha256
 base_channels = debian-9-pool-amd64
 repo_url = http://security-cdn.debian.org/debian-security/dists/stretch/updates/main/binary-amd64/
 
+[debian-9-amd64-uyuni-client-master]
+label    = debian-9-amd64-uyuni-client-master
+name     = Uyuni Master Client Tools for Debian 9 Stretch AMD64
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-9-pool-amd64
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/Debian9-Uyuni-Client-Tools/Debian_9/
+
+[debian-9-amd64-uyuni-client-stable]
+label    = debian-9-amd64-uyuni-client-stable
+name     = Uyuni Stable Client Tools for Debian 9 Stretch AMD64
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-9-pool-amd64
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian9-Uyuni-Client-Tools/Debian_9/
+
 [debian-10-pool-amd64]
```

Comment:
As in the end we are adding `-uyuni`  suffix to bootstrap to support Debian both now on Uyuni and on SUSE Manager later, may I suggest you duplocate Debian 9 and Debian 10 here, and you add `-uyuni` to the labels and names, just as we do for Ubuntu above?

This will also require a change at `susemanager/src/mgr_bootstrap_data.py` and the doc for PR.

Reply:
no probs will do that